Frameworks/Foundation/NSURLProtocol.mm

```diff
@@ -74,34 +74,13 @@ - (instancetype)initWithRequest:(NSURLRequest*)request
                  cachedResponse:(NSCachedURLResponse*)cachedResponse
                          client:(id<NSURLProtocolClient>)client {
     if (self = [super init]) {
-        _request.attach([request mutableCopy]);
-        _cachedResponse = cachedResponse;
-        _client = client;
+        _request = [request mutableCopy];
```

Comment:
they were properties, and one was working incorrectly and not taking a copy. seemed like a good idea to just pivot to autosynthesis